library/src/main/java/com/novoda/downloadmanager/lib/BatchQuery.java

```diff
@@ -44,6 +44,7 @@ String getSortOrder() {
 
         private static final String ORDER_BY_LIVENESS = "CASE " + DownloadContract.Batches.COLUMN_STATUS + " "
                 + "WHEN " + DownloadStatus.RUNNING + " THEN 1 "
+                + "WHEN " + DownloadStatus.BATCH_RUNNING + " THEN 2 "
```

Comment:
should the remaining status orders be bumped up? i.e. pending > 3, pausedbyapp > 4, etc.

Reply:
I wasn't sure about that change. I wanted to have the least impact on other parts of the code, so that BATCH_RUNNING is identical to PENDING (which it was), except that it can be differentiated at the app level to prevent "Queued" reappearing.

Reply:
But thinking about it, since it's for ordering, you're right, I should probably increase the next values instead, because we probably want batch downloads that are BATCH_RUNNING to be considered before those that are PENDING. Will fix.